content/response-to-reviewers.md

```diff
@@ -53,6 +53,12 @@ We found this approach allows users to modify source code at a scale that is mor
 What new features will be implemented?
 Do you plan to apply for a grant to further develop manubot? etc.
 
+We discuss this issue in [GH133](https://github.com/greenelab/meta-review/issues/133) and address it in [GH179](https://github.com/greenelab/meta-review/pull/179).
+Specifically, we added a sub-section to the titled "Future Manubot enhancements".
```

Comment:
`to the Discussion`

Reply:
It's now under the Manubot section

Reply:
But is there a word missing between `the` and `titled`?